### DISCUSSION TITLE
May I ask if Memospot supports the latest version(0.19.0) of the memos?

### DISCUSSION THREAD

**Discussion by yihanglee-cn** [memospot/memospot#22] [General]
As title

**Comment by lincolnthalles** [comment]
For macOS and Linux, it's better to wait a little more. The Memos front-end is not embedded in the server anymore and it requires some work that's already on the go.

But on Windows, Memospot 0.1.2 does support v0.19 as it is.

![image](https://github.com/lincolnthalles/memospot/assets/7476810/f2a6f297-c700-4417-a657-9315dd26b6b4)

You can simply run the updater script on Windows Powershell and it will backup previous data and replace Memos with v0.19.0.
```powershell
Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/lincolnthalles/memospot/main/memos-server-updater.ps1'))
```

Note that if you have a lot of notes and resources in the database, the first start may take some time and you should wait, even if you have to click "Try again" on Memospot loading screen from time to time.

Currently, Memos v0.19.0 have an issue with the version migration from v0.18. It's running super slow and takes a lot of resources until completion, [see this](https://github.com/usememos/memos/issues/2850). This only affects the first post-update startup. After it's done, it will start-up as fast as always.

**Reply by yihanglee-cn** [reply]
Thanks a lot.
